src/controllers/event/dashboard_event.controller.ts

```diff
@@ -0,0 +1,35 @@
+import { Router, Request, Response, NextFunction } from "express";
```

Comment:
since we're only having one route, I would suggest to move this route into the `event.controller.ts` file. 
hint : use this.path\top properly.

also, the response should be sent in the manner how the interface is defined. Modify the code by getting only the required fields which are present in the interface.



Make sure you test the endpoints.